src/main/java/edu/ucdavis/dss/ipa/api/components/course/CourseViewController.java

```diff
@@ -146,6 +146,24 @@ public void deleteCourse(@PathVariable long courseId, HttpServletResponse httpRe
 		courseService.delete(courseId);
 	}
 
+	@RequestMapping(value = "/api/courseView/schedules/{workgroupId}/{year}/courses", method = RequestMethod.PUT, produces="application/json")
+	@ResponseBody
+	public List<Long> deleteMultipleCourses(@PathVariable long workgroupId, @PathVariable long year, @RequestBody List<Long> courseIds, HttpServletResponse httpResponse) {
+		Schedule schedule = scheduleService.findByWorkgroupIdAndYear(workgroupId, year);
```

Comment:
Does Spring allow workgroupId or year to be null?

Reply:
Spring will throw a 400 if either are null.